mtp_noms_ops/assets-src/stylesheets/views/_checks.scss

```diff
@@ -73,3 +73,84 @@ table.mtp-table--review {
 .button-warning {
   @include button($red);
 }
+
+// TODO remove once new gov.uk styling implemented
```

Comment:
this is from the MoJ design system rather than the GDS one, right? i'd remove the TODO line cos the MoJ design system was an experiment and is allegedly abandoned.

Reply:
i.e. we'll still have to have these style after migrating

Reply:
It's whichever design system is used in: https://pm-intelligence-tool.herokuapp.com/delayed_capture_history_and_todo_02/index

Reply:
yeah i checked the prototype, it's not the GDS design system so we'll still have to have these style after migrating